css-three-column-layout/styles.css

```diff
@@ -0,0 +1,81 @@
+body {
+  background-color: #1b96d9;
+  color: white;
+  /* stylelint-disable-next-line font-family-no-missing-generic-family-keyword */
+  font-family: lato, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.container {
+  max-width: 800px;
+  margin: 0 auto;
+  flex-direction: row;
+  align-items: center;
+  padding-top: 100px;
+  justify-content: center;
+  justify-items: center;
+}
+
+.row {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  justify-items: center;
+}
+
+.column-full {
+  width: 100%;
+  margin: 10px;
+  padding-bottom: 20px;
+}
+
+.column-third {
+  flex-basis: 33%;
+  justify-items: center;
+  justify-content: center;
+  text-align: center;
+  align-items: center;
+  padding-left: 10px;
+  padding-right: 10px;
+}
+
+section {
+  width: 100;
+  justify-items: center;
+  justify-content: center;
+  text-align: center;
+}
+
+.opaque {
+  color: rgb(255 255 255 / 75%);
+  font-size: smaller;
+}
+
+h2 {
+  justify-content: center;
+  text-align: center;
+}
+
+img {
+  width: 100%;
+  padding: 20px;
+}
+
+div.circle {
+  border-radius: 50%;
+  background-color: white;
+  justify-content: center;
+  align-items: center;
+  width: 50%;
+  height: 150%;
```

Comment:
Percentages larger than 150% are bad news. SInce this needs to be a perfect circle, I would use a `px` measurement for width and height, since they need to be equivalent.